versioned_docs/version-V3/reference/Governance/governance-reference.md

```diff
@@ -8,7 +8,7 @@ tags: governance, documentation
 
 The Uniswap protocol is governed and upgraded by UNI token holders, using three distinct components; the UNI token, governance module, and Timelock. Together, these contracts allow the community to propose, vote, and implement changes to the uniswap protocol.
 
-Any addresses with more than 2.5M UNI delegated to it may propose governance actions, which contain finished, executable code. When a proposal is created, the community can cast their votes during a 3 day voting period. If a majority, and at least 4M votes are cast for the proposal, it is queued in the Timelock, and may be executed in a minimum of 2 days.
+Any addresses with more than 2.5M UNI delegated to it may propose governance actions, which contain finished, executable code. When a proposal is created, the community can cast their votes during a 7 day voting period. If a majority, and at least 40M votes are cast for the proposal, it is queued in the Timelock, and may be executed in a minimum of 2 days.
```

Comment:
it has to be at least 40m in favor (not total) and a simple majority

thanks for the fix!!

Reply:
Rephrased. Please review again and merge if ok. Thanks.